src/components/viewers/MultiFlatmap.vue

```diff
@@ -206,14 +235,8 @@ export default {
       if (force || (this.zoomLevel !== currentZoom)) {
         this.zoomLevel = currentZoom;
         flatmapImp.clearMarkers();
-        let markers = store.state.settings.markers
-        markers.forEach(id=>{
-          markerZoomLevels.map(el=>{
-            if (el.id === id && this.zoomLevel >= el.showAtZoom) {
-              flatmapImp.addMarker(id, "simulation");
-            }
-          })
-        })
+        let markers = store.state.settings.markers;
+        checkMarkersAtZoomLevel(flatmapImp, markers, this.zoomLevel);
```

Comment:
Based on the current code in `checkMarkersAtZoomLevel()`, it looks like this line could simply be replaced with:
```javascript
markers.forEach(id => {
  flatmapImp.addMarker(id, "simulation");
});
```

Reply:
No, it does not. checkMarersAtZoomLevel does not add any marker that fails the condition.

Reply:
Oops, my bad, I focussed on `foundInArray` and overlooked your `zoomLevel >= markerZoomLevels[i].showAtZoom` test!